packages/app/src/device-manager/services/device-manager.service.ts

```diff
@@ -150,8 +156,26 @@ export class DeviceManager {
     })
   }
 
-  @log("==== serial port: list ====")
+  @log("==== device manager: list ====")
   private getSerialPortList(): Promise<SerialPortInfo[]> {
     return SerialPort.list()
   }
+
+  private mountListeners(): void {
+    this.eventEmitter.on(
+      DeviceServiceEvent.DeviceInitializationFailed,
+      this.deviceInitializationFailedListener
+    )
+    this.eventEmitter.on(
+      DeviceServiceEvent.DeviceConnected,
+      this.deviceConnectedListener
+    )
+  }
```

Comment:
Wonder why additional function and not simply doing it in constructor? To describe it better why it is there?